bundle-konflux.Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Workaround for https://issues.redhat.com/browse/STONEBLD-2496
```

Comment:
does this has to be in the root? does it makes sense to place it in the /build dir along the others dockerfiles?